main.php

```diff
@@ -21,6 +21,16 @@
     exit(2);
 }
 
+function fillHeader($header)
+{
+    array_walk($header, function (&$value, $index) {
+        if (!$value) {
+            $value = "col_" . ($index + 1);
```

Comment:
nedáme těm `col_#` padding nulama? ale nebude to zpětně kompatibilní :-(

Reply:
proc? aby jsme udelali nejakou zpetne nekompatibilni zmenu? :D

Reply:
asi jen kvůli sortování v nějakým pitomým UIčku :-)

```
col_01
col_10
```
...